repository/Repository.go

```diff
@@ -562,3 +562,51 @@ func DoesColumnExistInTable(uow *UnitOfWork, tableName string, ColumnName string
 	//tableName := uow.DB.NewScope(rules).TableName() // rules --> model, need to send from client controller
 	return uow.DB.Dialect().HasColumn(tableName, ColumnName)
 }
+
+// CreateOrderByString get order by string
+func CreateOrderByString(orderByAttrs []string, validOrderByAttrs []string, orderByAttrAndDBCloum map[string][]string) (string, error) {
```

Comment:
What is the difference between this method and GetOrderBy?

Reply:
GetOrderBy return query processor while this method returns string. In reporting we require string input.
This method was previously in Base Service. I moved it to Repository to keep both the methods under one file.